src/test/java/org/prebid/server/bidder/invibes/InvibesBidderTest.java

```diff
@@ -278,6 +281,55 @@ public void makeHttpRequestsShouldCreateInvibesBidRequestWithCorrectParams() thr
                 .bidParamsJson(mapper.writeValueAsString(invibesBidParams))
                 .isTestBid(true)
                 .location(PAGE_URL)
+                .isAmp(false)
+                .gdpr(true)
+                .gdprConsent(StringUtils.EMPTY)
+                .invibBVLog(true)
+                .videoAdDebug(true)
+                .lid(BUYER_UID)
+                .bvid("test")
+                .width(String.valueOf(DEVICE_W))
+                .height(String.valueOf(DEVICE_H))
+                .build();
+
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(HttpRequest::getPayload)
+                .containsOnly(expectedRequest);
```

Comment:
There is no need to check for size if you use `.containsExactly` at the end

Reply:
I copied the test from above and changed only the check for `isAMP`. Should I resolve this?

Reply:
done for both

Reply:
Would be really nice.